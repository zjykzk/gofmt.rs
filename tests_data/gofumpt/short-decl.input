package p

var global = x

func f() {
	var local = x
	var local2, local3 = x, y

	var onlyType T

	var typeAndVar T = x

	var _ = unused

	var (
		aligned = x
		vars    = y
		here    = y
	)
}
