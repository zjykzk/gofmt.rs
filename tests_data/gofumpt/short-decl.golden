package p

var global = x

func f() {
	local := x
	local2, local3 := x, y

	var onlyType T

	var typeAndVar T = x

	_ = unused

	var (
		aligned = x
		vars    = y
		here    = y
	)
}
